packages/phone-number-privacy/signer/src/migrations/20200330212224_create-accounts-table.ts

```diff
@@ -0,0 +1,21 @@
+import * as Knex from 'knex'
+import { ACCOUNTS_COLUMNS, ACCOUNTS_TABLE } from '../database/models/account'
+
+export async function up(knex: Knex): Promise<any> {
+  // This check was necessary to switch from using .ts migrations to .js migrations.
+  if (!(await knex.schema.hasTable(ACCOUNTS_TABLE))) {
+    return knex.schema.createTable(ACCOUNTS_TABLE, (t) => {
+      t.string(ACCOUNTS_COLUMNS.address)
+        .notNullable()
+        .primary()
+      t.dateTime(ACCOUNTS_COLUMNS.createdAt).notNullable()
+      t.integer(ACCOUNTS_COLUMNS.numLookups).unsigned()
+      t.dateTime(ACCOUNTS_COLUMNS.didMatchmaking)
+    })
+  }
+  return new Promise((resolve) => resolve(null))
```

Comment:
Was this Promise wrapper necessary? Al async functions return promises by default, wondering if `return null` is equivalent here.

Reply:
Good call! Looks like it's not necessary